_posts/2015-07-21-the-epages-selenium-framework.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "The ePages Selenium Framework"
+date: "2015-07-21 17:06:00"
+icon: ship
+tags: selenium, test, continuous delivery
+categories: tech-stories
+authors: ["Jan M.", "Martin K.", "Kay A."]
+---
+
+With the help of Selenium we create and run automated test scripts for
```

Comment:
Since we assume in this article that the reader doesn't know what Selenium is, we should explain this here already. Suggestion: "With the help of the open source projcet Selenium we create..."